libs/blocks/tabs/tabs.js

```diff
@@ -68,6 +68,11 @@ function configTabs(config, rootElem) {
     const sel = rootElem.querySelector(id);
     if (sel) sel.click();
   }
+  const tabParam = new URLSearchParams(window.location.search).get('tab');
```

Comment:
please add unit test for this

Reply:
@robert-bogos left a comment on his PR https://github.com/adobecom/milo/pull/2652#issuecomment-2260744946

Also, we discourage requesting changes for unit test / coverage purposes. It's best to directly create a ticket so that this can be followed up on.